vulnfeeds/cves/versions_test.go

```diff
@@ -903,37 +991,45 @@ func TestExtractVersionInfo(t *testing.T) {
 				AffectedCommits:  []AffectedCommit(nil),
 				AffectedVersions: []AffectedVersion(nil),
 			},
-			expectedNotes: []string{},
-		},
-		// Currently planned outage: 2025-03-16 -> 2025-03-22
-		//		{
-		//			description:        "A CVE with a weird GitLab reference that breaks version enumeration in the worker",
-		//			inputCVEItem:       loadTestData2("CVE-2022-46285"),
-		//			inputValidVersions: []string{},
-		//			expectedVersionInfo: VersionInfo{
-		//				AffectedCommits:  []AffectedCommit{{Repo: "https://gitlab.freedesktop.org/xorg/lib/libxpm", Fixed: "a3a7c6dcc3b629d7650148"}},
-		//				AffectedVersions: []AffectedVersion{{Fixed: "3.5.15"}},
-		//			},
-		//			expectedNotes: []string{},
-		//		},
+			expectedNotes:     []string{},
+			disableExpiryDate: NO_EXPIRY,
+		},
+		{
+			description:        "A CVE with a weird GitLab reference that breaks version enumeration in the worker",
+			inputCVEItem:       loadTestData2("CVE-2022-46285"),
+			inputValidVersions: []string{},
+			expectedVersionInfo: VersionInfo{
+				AffectedCommits:  []AffectedCommit{{Repo: "https://gitlab.freedesktop.org/xorg/lib/libxpm", Fixed: "a3a7c6dcc3b629d7650148"}},
+				AffectedVersions: []AffectedVersion{{Fixed: "3.5.15"}},
+			},
+			expectedNotes:     []string{},
+			disableExpiryDate: time.Date(2025, 3, 22, 12, 30, 0, 0, time.Local),
+		},
 		{
 			description:  "A CVE with a different GitWeb reference URL that was not previously being extracted successfully",
 			inputCVEItem: loadTestData2("CVE-2021-28429"),
 			expectedVersionInfo: VersionInfo{
 				AffectedCommits:  []AffectedCommit{{Repo: "https://git.ffmpeg.org/ffmpeg.git", Fixed: "c94875471e3ba3dc396c6919ff3ec9b14539cd71"}},
 				AffectedVersions: []AffectedVersion{{LastAffected: "4.3.2"}},
 			},
+
+			disableExpiryDate: NO_EXPIRY,
 		},
 		{
 			description:  "A CVE with a configuration unsupported by ExtractVersionInfo and a limit version in the description",
 			inputCVEItem: loadTestData2("CVE-2020-13595"),
 			expectedVersionInfo: VersionInfo{
 				AffectedVersions: []AffectedVersion{{Introduced: "4.0.0", LastAffected: "4.2"}},
 			},
+
+			disableExpiryDate: NO_EXPIRY,
 		},
 	}
 
 	for _, tc := range tests {
+		if time.Now().Before(tc.disableExpiryDate) {
```

Comment:
should we specify timezone here?

Reply:
Probably not? I think the usecase here is for disabling for multiple days, so a few hours probably doesn't matter I think.